packages/common/src/types/gdrive.ts

```diff
@@ -5,12 +5,17 @@ export type ApiGdriveAccessTokenResponse = {
     results: string;
 };
 
+export type CustomLabel = {
```

Comment:
workaround for tsoa  https://lightdash.slack.com/archives/C02GQKJK84Q/p1699535082291029?thread_ts=1699534442.072269&cid=C02GQKJK84Q